diff --git a/nuget-support/tv/Microsoft.Common.targets/ImportAfter/Microsoft.NuGet.ImportAfter.targets b/nuget-support/tv/Microsoft.Common.targets/ImportAfter/Microsoft.NuGet.ImportAfter.targets
index 3aa22ba..ca7237c 100644
--- a/nuget-support/tv/Microsoft.Common.targets/ImportAfter/Microsoft.NuGet.ImportAfter.targets
+++ b/nuget-support/tv/Microsoft.Common.targets/ImportAfter/Microsoft.NuGet.ImportAfter.targets
@@ -11,7 +11,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <NuGetTargets Condition="'$(NuGetTargets)'==''">$(MSBuildExtensionsPath)\Microsoft.NuGet.targets</NuGetTargets>
+    <NuGetTargets Condition="'$(NuGetTargets)'==''">$(MSBuildExtensionsPath)\Microsoft\NuGet\Microsoft.NuGet.targets</NuGetTargets>
   </PropertyGroup>
   <Import Condition="Exists('$(NuGetTargets)')" Project="$(NuGetTargets)" />
   <!-- Import NuGet.targets for Restore -->
