From c1ec6ffdeb85e7fc1f4652b85b721b084f0ae97d Mon Sep 17 00:00:00 2001
From: dseefeld <dseefeld@microsoft.com>
Date: Sat, 20 Jun 2020 18:54:24 -0400
Subject: [PATCH] Conditionally include PrecompileScript

---
 .../ArgumentForwarding.Tests/ArgumentForwarding.Tests.csproj    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Tests/ArgumentForwarding.Tests/ArgumentForwarding.Tests.csproj b/src/Tests/ArgumentForwarding.Tests/ArgumentForwarding.Tests.csproj
index da033cd6b..ab7ff3133 100644
--- a/src/Tests/ArgumentForwarding.Tests/ArgumentForwarding.Tests.csproj
+++ b/src/Tests/ArgumentForwarding.Tests/ArgumentForwarding.Tests.csproj
@@ -11,7 +11,7 @@
     <ProjectReference Include="..\Microsoft.NET.TestFramework\Microsoft.NET.TestFramework.csproj" />
   </ItemGroup>
 
-  <Target Name="PrecompileScriptBuild" BeforeTargets="Build" Condition=" '$(IsCrossTargetingBuild)' != 'true' ">
+  <Target Name="PrecompileScriptBuild" BeforeTargets="Build" Condition=" '$(IsCrossTargetingBuild)' != 'true' and '$(DotNetBuildFromSource)' != 'true'">
     <MSBuild
       BuildInParallel="False"
       Projects="../ArgumentsReflector/ArgumentsReflector.csproj"
-- 
2.25.4

