From 36233ce4ad5492b6022b59a1fec876800d6608a9 Mon Sep 17 00:00:00 2001
From: Chris Rummel <crummel@microsoft.com>
Date: Tue, 8 Sep 2020 12:20:40 -0500
Subject: [PATCH] Use source-built MSBuild for F#.

---
 eng/Versions.props | 1 +
 1 file changed, 1 insertion(+)

diff --git a/eng/Versions.props b/eng/Versions.props
index 7c7c0a349..cc43f479f 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -97,6 +97,7 @@
     <MicrosoftBuildFrameworkVersion>$(MicrosoftBuildFrameworkPackageVersion)</MicrosoftBuildFrameworkVersion>
     <MicrosoftBuildUtilitiesCoreVersion>$(MicrosoftBuildUtilitiesCorePackageVersion)</MicrosoftBuildUtilitiesCoreVersion>
     <MicrosoftBuildTasksCoreVersion>$(MicrosoftBuildTasksCorePackageVersion)</MicrosoftBuildTasksCoreVersion>
+    <FSharpBuildVersion>$(MicrosoftBuildPackageVersion)</FSharpBuildVersion>
   </PropertyGroup>
   <PropertyGroup>
     <!-- Dependencies from https://github.com/dotnet/templating -->
-- 
2.25.4

